admin/app/indexes/ContentApiTagsEnumerator.scala

```diff
@@ -39,7 +37,7 @@ object ContentApiTagsEnumerator extends Logging {
   }
 
   def enumerateTagType(tagType: String) = enumeratePages { page =>
-    getResponse(ContentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
+    contentApiClient.getResponse(contentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
```

Comment:
I mean the `contentApiClient.getResponse(contentApiClient.someQuery)` rather than something like `contentApiClient.someQuery.getResponse`